cmd/util/cmd/rollback-executed-height/cmd/rollback_executed_height_test.go

```diff
@@ -95,8 +95,27 @@ func TestReExecuteBlock(t *testing.T) {
 
 		require.NoError(t, err)
 
+		batch := bstorage.NewBatch(db)
+
 		// remove execution results
 		err = removeForBlockID(
+			batch,
+			headers,
+			commits,
+			txResults,
+			results,
+			chunkDataPacks,
+			myReceipts,
+			events,
+			serviceEvents,
+			header.ID(),
+		)
+
+		require.NoError(t, err)
+
+		// remove again, to make sure missing entires are handled properly
```

Comment:
To ensure missing entries are handled properly, here we make the tests to remove the result twice and flush once. But I don't think the actual code would be run like this. More likely that we will run the command line twice to remove results for the same block, which means we will run remove and flush twice for the same block, instead of remove the result twice and flush once. 

So it's better adjust the tests accordingly:
```
batch := bstorage.NewBatch(db)
err := removeForBlockID(batch, ...)
require.NoError(t, err)

err = batch.Flush()
require.NoError(t, err)

// remove again for the same block, it should not return any error if the results
// have been removed already.
batch2 := bstorage.NewBatch(db)
err := removeForBlockID(batch2, ...)
require.NoError(t, err)

err = batch2.Flush()
require.NoError(t, err)
```

Reply:
Interesting idea, I have made it test both - so remove twice, flush and than remove again. This should cover all cases.